docs/04-FunctionalTests.md

```diff
@@ -250,10 +267,43 @@ error_level: "E_ALL & ~E_STRICT & ~E_DEPRECATED"
 
 `error_level` can also be set globally in `codeception.yml` file.
 
+## Accessing external URL's with Guzzle
```

Comment:
No, it's not possible. Cookbooks contain current solution for very specific problem, while documentation is a general, constantly updated. You explain a very specific problem, and as I said this is not about functional tests at least in the term of Codeception. Also, functional tests are supposed to test the system from inside, not outside, so this example is not applicable here .

Btw, instead of using Guzzle you can use PhpBrowser for the same case (which was already explained in the previous chapter). 

Please remove that part from this PR and I will merge it. Thanks